.circleci/config.yml

```diff
@@ -46,8 +46,8 @@ workflows:
       - run:
           name: "tests"
           script: test
-  test-types:
-    jobs:
-      - run:
-          name: "test types"
-          script: test:types
+  # test-types:
+  #   jobs:
+  #     - run:
+  #         name: "test types"
+  #         script: test:types
```

Comment:
We definitely don't want to disable the test:types Why was it failing?

Reply:
I'm getting this error when I run `yarn test:types`:

```
yarn run v1.21.1
$ yarn workspaces run test:types

> effection
$ dtslint types --localTs node_modules/typescript/lib --expectOnly

Error: Errors in typescript@local for external dependencies:
../index.d.ts(3,15): error TS2300: Duplicate identifier 'Operation'.
../index.d.ts(5,8): error TS2300: Duplicate identifier 'OperationFn'.
../index.d.ts(7,8): error TS2300: Duplicate identifier 'Controller'.

    at /Users/minkimcello/Projects/thefrontside/effection.js/node_modules/dtslint/bin/index.js:193:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/minkimcello/Projects/thefrontside/effection.js/node_modules/dtslint/bin/index.js:6:58)
```

Reply:
@minkimcello you should never disable tests unless you got everyone's buy-in to do it.

Reply:
@taras oh okay

Reply:
@cowboyd Should I have not resolved this myself?